pkg/helm/charts/tests/securedclusterservices/testdata/helmtest/scanner-v4.test.yaml

```diff
@@ -195,7 +195,7 @@ tests:
     imagePullSecrets.allowNone: true
   expect: |
     .deployments["sensor"].spec.template.spec.containers[0].env[] |
-      select(.name == "ROX_SCANNER_V4_GRPC_ENDPOINT") | assertThat(.value == "scanner-v4-indexer.custom-ns.svc:8443")
+      select(.name == "ROX_SCANNER_V4_INDEXER_GRPC_ENDPOINT") | assertThat(.value == "scanner-v4-indexer.custom-ns.svc:8443")
```

Comment:
I feel we should drop `_GRPC_` from this name. We don't have plans to use something else for now, but not specifying the protocol we use to connect to the specified address (or "endpoint") makes more sense to me.

Reply:
I think we added gRPC because Scanner-slim did that, but you're right. The other endpoints don't say HTTP, so why put the protocol. Removed